### PR TITLE
Update Workshop Framework

### PR DESCRIPTION
**Pull request by dm703**
Add bethesda.net URL location
Add currently known incompatibilities

### CODE REVIEW

**Comment by dm703** [comment]
Sorry, this is a bit of a mess (as Github Desktop has dragged in the other stuff relating to my fork). The only commit that should matter is 'Update Workshop.esm'
I guess I should have simply deleted the current fork and started over with a new fork for this one.

File: masterlist.yaml [hunk 1]
```diff
@@ -1562,9 +1562,20 @@ plugins:
         util: 'FO4Edit v4.0.3h (Hotfix 1)'
 
   - name: 'WorkshopFramework.esm'
-    url: [ 'https://www.nexusmods.com/fallout4/mods/35004/' ]
+    url:
+      - link: 'https://www.nexusmods.com/fallout4/mods/35004/'
+        name: 'Workshop Framework (Nexus Mods)'
+      - link: 'https://bethesda.net/en/mods/fallout4/mod-detail/4081586'
```
Comment (sibir-ine) [inline masterlist.yaml]: Bethesda.net URLs can have trailing slashes:

```suggestion
      - link: 'https://bethesda.net/en/mods/fallout4/mod-detail/4081586/'
```

File: masterlist.yaml [hunk 2]
```diff
@@ -1562,9 +1562,20 @@ plugins:
...
+        name: 'Workshop Framework (Bethesda.net Mods)'
     group: *frameworksGroup
     after: [ 'Unofficial Fallout 4 Patch.esp' ]
+    inc:
+      - name: 'Moddable Plasma Caster.esp'
+        detail: 'See https://wiki.simsettlements2.com/getting-started/known-issues'
```
Comment (sibir-ine) [inline masterlist.yaml]: Bare links should be avoided. It also appears that you can link to specific sections of that page:

```suggestion
        detail: 'See Sim Settlements 2's [compatibility notes](https://wiki.simsettlements2.com/getting-started/known-issues#the-p-113-a-highly-moddable-heavy-plasma-caster) on Workshop Framework'
```

File: masterlist.yaml [hunk 3]
```diff
@@ -1562,9 +1562,20 @@ plugins:
...
+      - name: 'NeutralVertibirdsEN.esp'
+        detail: 'See https://wiki.simsettlements2.com/getting-started/known-issues'
```
Comment (sibir-ine) [inline masterlist.yaml]:
```suggestion
        detail: 'See Sim Settlements 2's [compatibility notes](https://wiki.simsettlements2.com/getting-started/known-issues#neutral-vertibirds) on Workshop Framework'
```

File: masterlist.yaml [hunk 4]
```diff
@@ -1562,9 +1562,20 @@ plugins:
...
+      - name: 'NeutralVertibirdsDE.esp'
+        detail: 'See https://wiki.simsettlements2.com/getting-started/known-issues'
```
Comment (sibir-ine) [inline masterlist.yaml]:
```suggestion
        detail: 'See Sim Settlements 2's [compatibility notes](https://wiki.simsettlements2.com/getting-started/known-issues#neutral-vertibirds) on Workshop Framework'
```

**Review by sibir-ine — CHANGES_REQUESTED** [review]
For keeping your fork clean, you can indeed just delete your fork & fork the repo again. You can also make your changes on separate branches so that they can be deleted when they're merged & just pull the merged changes from our repo to your main branch without any merge commits. You can also reset or rebase your main branch so that it's even with our repo after your changes are merged.

File: masterlist.yaml (hunk 1 above)
Comment (dm703) [reply masterlist.yaml]: Sorry, you told me that before. I will eventually learn. :-)

File: masterlist.yaml (hunk 2 above)
Comment (dm703) [reply masterlist.yaml]: Thanks. I puzzled over how to represent it.  Now that I have a working example of the syntax, I can use it.

File: masterlist.yaml [hunk 5]
```diff
@@ -1562,9 +1562,20 @@ plugins:
...
+        name: 'Workshop Framework (Bethesda.net Mods)'
     group: *frameworksGroup
     after: [ 'Unofficial Fallout 4 Patch.esp' ]
+    inc:
+      - name: 'Moddable Plasma Caster.esp'
+        detail: 'See Sim Settlements 2's [compatibility notes](https://wiki.simsettlements2.com/getting-started/known-issues#the-p-113-a-highly-moddable-heavy-plasma-caster) on Workshop Framework'
+      - name: 'NeutralVertibirdsEN.esp'
+        detail: 'See Sim Settlements 2's [compatibility notes](https://wiki.simsettlements2.com/getting-started/known-issues#neutral-vertibirds) on Workshop Framework'
+      - name: 'NeutralVertibirdsDE.esp'
+        detail: 'See Sim Settlements 2's [compatibility notes](https://wiki.simsettlements2.com/getting-started/known-issues#neutral-vertibirds) on Workshop Framework'
```
Comment (sibir-ine) [inline masterlist.yaml]: Sorry about that, I forgot to escape the single quotes:

```suggestion
        detail: 'See Sim Settlements 2''s [compatibility notes](https://wiki.simsettlements2.com/getting-started/known-issues#the-p-113-a-highly-moddable-heavy-plasma-caster) on Workshop Framework'
      - name: 'NeutralVertibirdsEN.esp'
        detail: 'See Sim Settlements 2''s [compatibility notes](https://wiki.simsettlements2.com/getting-started/known-issues#neutral-vertibirds) on Workshop Framework'
      - name: 'NeutralVertibirdsDE.esp'
        detail: 'See Sim Settlements 2''s [compatibility notes](https://wiki.simsettlements2.com/getting-started/known-issues#neutral-vertibirds) on Workshop Framework'
```

File: masterlist.yaml (hunk 1 above)
Comment (sibir-ine) [reply masterlist.yaml]: No worries!

File: masterlist.yaml (hunk 5 above)
Comment (dm703) [reply masterlist.yaml]: We will get there.  thanks!

**Review by sibir-ine — APPROVED** [review]
Thanks!

**Pull request merged**